lib/migration.pm

```diff
@@ -151,7 +151,9 @@ sub remove_dropped_modules_packages {
 sub disable_installation_repos {
     if (is_s390x) {
         my $repos = script_output("zypper lr -u | awk '/ftp:.*?openqa.suse.de|10.145.10.207|10.160.0.100/ {print \$1}'");
```

Comment:
It seems the output with BR which caused the failure, could we update regexp to replace BR with space then don't need add code to deal with each line?

Reply:
As the output , we need remove those three repos. https://openqa.suse.de/tests/14785327#step/patch_sle/20
Is it expected? @hjluo

Reply:
Is remove BR, it will change to '123'. 
 zypper -n mr -d 123
Repository 123 not found.

Reply:
Shouldn't remove but replace BR with space, that output should be '1 2 3'.

Reply:
https://openqa.suse.de/tests/14800054#step/patch_sle/16